docs/en_US/guide/sinks/builtin/redis.md

```diff
@@ -4,16 +4,17 @@ The sink will publish the result into redis.
-| field         | true     | This field must exist and be of type string. Otherwise, use the field character as the key. Note: Do not use a data template to configure this value                                                 |
-| dataType      | false    | The default Redis data type is string. Note that the original key must be deleted after the Redis data type is changed. Otherwise, the modification is invalid. now only support "list" and "string" |
-| expiration    | false    | Timeout duration of Redis data. This parameter is valid only for string data in seconds. The default value is -1                                                                                     |
-| rowkindField  | true     | Specify which field represents the action like insert or update. If not specified, all rows are default to insert.                                                                                   |
+| Property name | Optional | Description                                                                                                                                                                                                           |
+|---------------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| addr          | false    | The addr of the Redis,example: 10.122.48.17:6379                                                                                                                                                                      |
+| password      | true     | The Redis login password                                                                                                                                                                                              |
+| db            | false    | The database of the Redis,example: 0                                                                                                                                                                                  |
+| key           | false    | Select one of the Key, Key and field of Redis data and give priority to field ; users can also set multiple key-value pair in result map by using the ``fields`` property                                             |
+| field         | true     | This field must exist and be of type string. Otherwise, use the field character as the key. Note: Do not use a data template to configure this value                                                                  |
+| fields        | true     | The fields to be inserted/updated to. The result map should have these fields and corresponding values. If one field not exist in result map, will be ignored. If not set, will use ``key`` or ``field`` as redis key |
```

Comment:
fields should keep the same meaning as tdengine/sql. It is used to filter the value payload, not the key.